tensorflow/contrib/seastar/seastar_tag_factory.cc

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017, Alibaba Inc.
+// All right reserved.
+//
+// Author: Liangbin LI <rangeng.llb@taobao.com>
+// Created: 2017/11/23
+// Description
+
+#include <assert.h>
```

Comment:
Better change to ``<cassert>`` as this is C++.

Reply:
useless code , removed